Python Tutorial Exercises(Week 3)/#7.py

```diff
@@ -0,0 +1,63 @@
+'''
+Solve Week 2 exercises using while loops instead of for loops
+'''
+#1
+x = [["this", "is", "a", "nested"], ["list", "that"], ["will", "be", "printed", "out"]]
+#                0                         1                         2
+nested_list_one = 0
+nested_list_two = 0
+#len(x)=3 and nested_list_one will never greater than 3 becuse there only 3 list in the x list[0,1,2]. So we use '<' in here.
+while nested_list_one < len(x):
+    #nested_list_two = [ [0,1,2,3], [0,1], [0,1,2,3]  ] . 
+    #len(x[nested_list_one]) = 4444(nested list 1 have 4 items and so on),22,4444.
+    #nested_list_two will never greater than nested_list_one, so use '<' in here.
```

Comment:
These comments aren't making a lot of sense...
`len(x[nested_list_one]) = 4444`?
`nested_list_two will never greater than nested_list_one` This isn't true and that is not what the comparison is that you have in code